src/styles/global.css

```diff
@@ -57,3 +57,9 @@ html[data-theme="dark"] {
 .active-nav {
   @apply underline decoration-wavy decoration-2 underline-offset-4;
 }
+
+/* Source: https://piccalil.li/blog/a-more-modern-css-reset/ */
+/* Anything that has been anchored to should have extra scroll margin */
+:target {
+  scroll-margin-block: 5ex;
```

Comment:
For this project, I think it's better to have `1rem` instead of `5ex`. 
Is there any other specific reason to use `5ex` instead of other values? 
Maybe I might have missed something, idk 🤷

Reply:
Yeah actually I forgot why `ex` over `rem`, It was YT video where the blog author explained it (also couldn't find the video). But yeah `rem` should be equally good in my opinion 👍🏻